Not applicable on Linux, saves having to add this module as a separate aport

--- a/META.json
+++ b/META.json
@@ -42,7 +42,6 @@
             "CPAN::Meta::Requirements" : "2.130",
             "CPAN::Meta::YAML" : "0",
             "Command::Runner" : "0.100",
-            "Darwin::InitObjC" : "0",
             "ExtUtils::Config" : "0",
             "ExtUtils::Helpers" : "0",
             "ExtUtils::Install" : "2.20",
--- a/lib/App/cpm/CLI.pm
+++ b/lib/App/cpm/CLI.pm
@@ -20,7 +20,6 @@ use CPAN::Meta;
 use Command::Runner;
 use Config;
 use Cwd ();
-use Darwin::InitObjC;
 use File::Copy ();
 use File::Path ();
 use File::Spec;
@@ -366,8 +365,6 @@ sub cmd_install {
 sub install {
     my ($self, $master, $worker, $num) = @_;
 
-    Darwin::InitObjC::maybe_init();
-
     my @task = $master->get_task;
     Parallel::Pipes::App->run(
         num => $num,
